.github/pull_request_template.txt

```diff
@@ -0,0 +1,18 @@
+## Problem
+- Describe the problem this PR solves here.
+- If you're fixing an issue, say "Fixes #X".
+
+## Solution
+- Describe the solution used to overcome the problem.
+
+## Additional Information
+- Any additional information needed to better understand this PR.
+
+## Breaking Changes (if applicable)
+- What changes were introduced that would break backwards compatibility?
+- How would a user modify their code to adjust for these breaking changes?
+
+## Checklist (if applicable)
+- [ ] Tests
```

Comment:
Is this "adding tests" or "running tests"?

Reply:
Adding tests (I'll clarify in template)